addons/plan_importer/functions/fn_addActions.sqf

```diff
@@ -20,7 +20,7 @@ private _parentAction = [
 	"AET_Plan_Actions",
 	"Plan Actions",
 	"",
-	{},
+	{},                     // could just check for the main setting here: < missionNamespace getVariable [QSET(enabled), false]; > - no need for any of the add/remove action stuff.
```

Comment:
This code block here is the condition for the parent action.

IF you check here for the setting being enabled or not, it will ether show or not show up

Reply:
Hmmmmm, could potentially do that, but I remember you saying it is better than having actions hidden if they are not in use?

If I misunderstood then can prob do that.